topics/synthetic-biology/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: synthetic-biology
+type: use
+title: Title of the topic
+summary: Summary of the topic
+requirements:
+- type: internal
+  topic_name: introduction
+docker_image: ''
+maintainers:
+- maintainers
```

Comment:
This file needs to be filled out, the maintainer list should include some names (perhaps all of yours), and the title/summary should be filled out with some text.

Reply:
Ok, Done.